### PR TITLE
cmds/core/xargs: minimal xargs, just passing args

### PR DESCRIPTION
**Pull request by binjip978**
Parsing is simplified by strings.Fields, so no quotes support.

### CODE REVIEW

**Review by rminnich — COMMENTED** [review]
interestingly, xargs with no args is cat (from experiment)

might want to make the run function take a variadic?

**Comment by binjip978** [comment]
> interestingly, xargs with no args is cat (from experiment)

Interesting because both from Linux man page `executes the command (default is /bin/echo)` and macOS `If utility is omitted, echo(1) is used.` In what system it use cat as default?

> might want to make the run function take a variadic?

It can be done, but os.Args is kinda variadic already. So maybe  I missing how it will improve any scenario?

File: cmds/core/xargs/xargs.go [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+	"strings"
+)
+
+func main() {
+	if err := run(os.Stdin, os.Stdout, os.Stderr, os.Args[1:]); err != nil {
+		log.Fatal(err)
+	}
+}
+
+func run(stdin io.Reader, stdout, stderr io.Writer, args []string) error {
```
Comment (rminnich) [inline cmds/core/xargs/xargs.go:22]: I was things args could be ...string, not []string, since they are allowed to be empty.

**Review by rminnich — APPROVED** [review]
my comment is really not that important, I leave the decision to you.

File: cmds/core/xargs/xargs.go (hunk 1 above)
Comment (binjip978) [reply cmds/core/xargs/xargs.go]: yeah test will look better, updated, thx.

**Review by rminnich — APPROVED** [review]

**Review by rminnich — APPROVED** [review]

**Pull request merged**